website/content/en/docs/building-operators/golang/references/env-test-setup.md

```diff
@@ -10,29 +10,20 @@ This document describes how to configure the environment for the [controller tes
 
 ## Installing prerequisites
 
-[Envtest][envtest] requires that `kubectl`, `api-server` and `etcd` be present locally. You can use this [script][script] to download these binaries into the `testbin/` directory. It may be convenient to add this script your Makefile as follows:
+[Envtest][envtest] requires that `kubectl`, `api-server` and `etcd` be present locally. You can use this [script][script] to download these binaries into the `testbin/` directory which will be created in your project. Update your Makefile by replacing your `test` target with: 
 
 ```sh
-# Setup binaries required to run the tests
-# See that it expects the Kubernetes and ETCD version
-K8S_VERSION = v1.18.2
-ETCD_VERSION = v3.4.3
-testbin:
-	curl -sSLo setup_envtest.sh https://raw.githubusercontent.com/kubernetes-sigs/kubebuilder/master/scripts/setup_envtest_bins.sh 
-	chmod +x setup_envtest.sh
-	./setup_envtest.sh $(K8S_VERSION) $(ETCD_VERSION)
+# Run tests
+ENVTEST_ASSETS_DIR=$(shell pwd)/testbin
+test: generate fmt vet manifests
+        mkdir -p ${ENVTEST_ASSETS_DIR}
+        test -f ${ENVTEST_ASSETS_DIR}/setup-envtest.sh || curl -sSLo ${ENVTEST_ASSETS_DIR}/setup-envtest.sh https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/master/hack/setup-envtest.sh
+        source ${ENVTEST_ASSETS_DIR}/setup-envtest.sh; fetch_envtest_tools $(ENVTEST_ASSETS_DIR); setup_envtest_env $(ENVTEST_ASSETS_DIR); go test ./... -coverprofile cover.out
 ```
 
+Also, it is recommended add into the `.gitignore` a new line with `testbin/*` for you do not commit these binaries. 
 
-The above script sets these environment variables to specify where test binaries can be found. In case you would like to not use the script then, is possible to do the same configuration to inform the path of your binaries: 
-
-```shell
-$ export TEST_ASSET_KUBECTL=<kubectl-bin-path>
-$ export TEST_ASSET_KUBE_APISERVER=<api-server-bin-path>
-$ export TEST_ASSET_ETCD=<etcd-bin-path>
-``` 
-
-See that the environment variables also can be specified via your `controllers/suite_test.go` such as the following example. 
+See that you can also use your own binaries and change the location via setting up the following environment variables in your `controllers/suite_test.go`: 
```

Comment:
I think I would suggest removing this and everything below.

The above description is short, straightforward, descriptive, and can get users up and running. I think that's all we need.